### PR TITLE
Move SwissProt.__init__ selftest to doc test.

### PR DESCRIPTION
**Pull request by vincentdavis**
See issue issue #820

### CODE REVIEW

File: Bio/SwissProt/__init__.py [hunk 1]
```diff
@@ -62,6 +62,25 @@ class Record(object):
         - seqinfo           tuple of (length, molecular weight, CRC32 value)
         - sequence          The sequence.
 
+        Example:
```
Comment (peterjc) [inline Bio/SwissProt/__init__.py]: That should be about four spaces further to the left I think, likewise the doctest which follows. Double check in your favourite reStructuredText editor - or an online one?

**Comment by vincentdavis** [comment]
@peterjc I fixed the indentation. I am not really sure how the "review requested changes" works. Are you able to propose changes that I can merge back in? I guess I need to learn more about that.

**Comment by peterjc** [comment]
Github have some new review/comment system which I'd not tried before - not sure how it is meant to work, but the change is what I had in mind - thanks!

**Comment by peterjc** [comment]
Apologies, I realised after the merge why the coverage had not improved - we currently whitelist all the doctests in ``Tests/run_tests.py`` -  see 05d196251048443591d4c855fabdfde900b28a37

**Pull request merged**